video_xblock/static/js/videojs_event_plugin.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * This part is responsible for tracking video player events.
+ * List of events:
+ * - onReady
+ * - onPlay
+ * - onPause
+ * - onEnded
+ * - onSeek
+ * - onSpeedChange
+ *
+ */
+(function () {
+
+    "use strict";
+    function XBlockEventPlugin(options) {
+
+        var previousTime = 0;
+        var currentTime = 0;
+
+        this.events = [
+            'onReady',
+            'onPlay',
+            'onPause',
+            'onEnded',
+            'onSpeedChange',
+             'onSeek',
+            /*TODO
```

Comment:
I understand why you want to comment these events. But the reason is not clear from the code.

If this plugin works with these lines uncommented don't comment these events.
If it doesn't work, add a note when to uncomment these events.